Extension/src/Debugger/attachToProcess.ts

```diff
@@ -108,6 +109,16 @@ export class RemoteAttachPicker {
     private getRemoteProcessCommand(): string {
         let innerQuote: string = `'`;
         let outerQuote: string = `"`;
+        let substitutionBegin: string = `$(`;
+        let substitutionEnd: string = `)`;
+        let escapedQuote: string = `\\\"`;
+
+        let settings: CppSettings = new CppSettings();
+        if (settings.useBacktickCommandSubstitution) {
+            substitutionBegin = `\``;
+            substitutionEnd = `\``;
+            escapedQuote = `\"`;
```

Comment:
Does backtick substitution mean that we don't need to escape the `\` symbol?

Reply:
Originally with the escaped `\` the command failed with an unmatched `"` error. I've just looked into this again and it seems to actually be a quirk in csh where stuff in single quotes doesn't get expanded: [Stack Overflow answer](https://stackoverflow.com/questions/10761838/getting-unmatched-on-a-simple-csh-ssh-script)